microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshot-schema.md

```diff
@@ -303,6 +303,66 @@ The first number in the group of numbers for an edge in the `edges` array corres
 | Shortcut | An easier-to-read representation of some other path. This type is rarely used. For example, if you use `Function.prototype.bind` to create a bound function with some bound arguments, V8 creates a `JSBoundFunction`, which points to a `FixedArray` (an internal type), which points to each bound argument. When producing a snapshot, V8 adds a shortcut edge from the bound function directly to each bound argument, bypassing the `FixedArray`. |
 | Element | Object properties where the key is a number. |
 
+#### locations
+
+The `locations` array, which is at the top level of the `.heapsnapshot` data, contains information about the location of HeapSnapshot nodes.
```

Comment:
```suggestion
The `locations` array, which is at the top level of the `.heapsnapshot` data, contains information about the location of `HeapSnapshot` nodes.
```
Research notes:

is "HeapSnapshot" a real thing?  Not able to confirm.  FTS in rendered page 
https://github.com/christyycchen/edge-developer/blob/patch-1/microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshot-schema.md
finds:   
.heapsnapshot   16x
heap snapshot    12x
heapsnapshot  18x  therefore two are not .heapsnapshot, they are:  the present "HeapSnapshot" and "the HeapSnapshotGenerator class in heap-snapshot-generator.h." which is tagged to link to: 
https://chromium.googlesource.com/external/v8/+/master/src/heap-snapshot-generator.h#142
which raises question is there really such a class as HeapSnapshotGenerator?  Line 142 that the link scrolls to, is NOT HeapSnapshotGenerator, but rather, is just HeapSnapshot.  

Link text should accurately represent where you end up.  The link text says HeapSnapshotGenerator, yet actually goes to HeapSnapshot - confusing/disorienting.  HeapSnapshotGenerator is line 522, not 142.   So the .md has an error: link text says HeapSnapshotGenerator, yet link goes to line 142, which is HeapSnapshot class, not HeapSnapshotGenerator.  Recommend tagging both links:

142 = class HeapSnapshot
522 = class HeapSnapshotGenerator

I might tag this up in VS Code and then enter a Comment to implement.